Client/Assets/Search/SearchPlugins/wikipedia-NN.xml

```diff
@@ -12,12 +12,12 @@
 </Url>
 <Url type="text/html" method="GET" template="https://nn.wikipedia.org/wiki/Spesial:Søk">
   <Param name="search" value="{searchTerms}"/>
-  <Param name="sourceid" value="Mozilla-search"/>
```

Comment:
I am wondering if there would be a more elegant way to exclude `sourceid` from all the wikipedia entries 🤔 Maybe something that lives in the code, not in the xml?

Reply:
Similar to the [in the other comment](https://github.com/ecosia/ios-browser/pull/596#discussion_r1383380940), there is no guarantee that the values that Firefox puts as for XML params are the same. In this PR I did remove Param names such us `sourceId` and `client`, both eventually referring to FF.

Reply:
Adding to @ecotopian's point, I noticed that `OpenSearchParser` has a logic that goes over `let urlIndexers = docIndexer.children(tag: "Url")`, which would be those params, and then appends them to the search template url string. I believe we could catch the `sourceid` name or the value itself inside an _if_ and not add them to the template via code, what do you think?

I like that suggestion since it's cleaner and would potentially make our lives easier when updating as well. It seems to be safe with it being the only place used to read those files, but if there is indeed no guarantee like you mentioned then I'd agree it might be better to change directly on the xmls.

Reply:
Cool, thanks for the explanation! 

I'd vote for doing it in code if safe, at least for the cases that require a change in a large number of files, like the `sourceId`, but I understand your point and will leave the final decision for you 🙂

Reply:
I can understand both reasonings. In order to keep the change set small, I'd tend to filter the `sourceId` in code - this would kind of act as a catch all mechanism, and also scale in the unlikely case that there are new wikipedia countries (which probably does not happen)

But I also see the concerns of keeping changes to the business logic codebase minimal which would favour removing the params from the xml itself. 

I think there's no clear right or wrong. Both approaches will work. I let you decide.

Reply:
I'll keep it as democratic as possible as there is no personal implication here! 😅.

So if you both think that going for excluding `sourceId`, `client`, `pc` and `form` (as I'd like to stress that, for now, we have different parameters in different search engines):
- `sourceId` for Wikipedia
- `client` for Google
- `pc` and `form` for Bing

I will proceed with excluding all these fields at the codebase level.